test/gtest/ucs/test_config.cc

```diff
@@ -329,8 +329,15 @@ class test_config : public ucs::test {
             delete [] m_value;
         }
 
-        ucs_status_t set(const char *name, const char *value) {
-            return ucs_config_parser_set_value(&m_opts, car_opts_table, NULL,
+        ucs_status_t set(const char *name, const char *value)
```

Comment:
since prefix is optional arg, maybe put it last with default value:
```C++
ucs_status_t set(const char *name, const char *value, const char *prefix = NULL)
```

Reply:
I prefer don't change logical ordering of args